product_docs/docs/pgd/5.7/reference/nodes-management-interfaces.mdx

```diff
@@ -345,14 +345,14 @@ from the node.
 
 You can call the function from any active node in the PGD group, including the node that you're removing.
 
-Executing the parting from the node being removed does run the risk of incorrectly reporting, or never reporting, the status of the removal.
+Executing parting from the node being removed runs the risk of incorrectly reporting, or never reporting, the status of the removal.
 This is because in the process of being removed, communications are cut off from the rest of the cluster.
-While the removal may succeed, there is no way to inform the node that issued the command failed or succeeded on the other nodes.
+While the removal may succeed, there's no way to inform the node that issued the command that it failed or succeeded on the other nodes.
 The function can't be set to wait for completion either, for the same reason.
 
-Once a node has parted itself, it cannot part other nodes in the cluster as it is no longer part of the cluster.
+Once a node has parted itself, it can't part other nodes in the cluster as it's no longer part of the cluster.
 
-We recommend avoiding using nodes to part themselves from the cluster and, instead, performing part node operations from a node that can wait for completion and check the cluster status after the operation is complete.
+We recommend avoiding using nodes to part themselves from the cluster and, instead, performing node parting operations from a node that can wait for completion. Then check the cluster status after the operation is complete.
```

Comment:
@djw-m Some of these wording changes on the various forms of the word "part" were for readability but I did want to check whether these changes weren't using the vocabulary incorrectly.